apps/modernization-api/src/test/java/gov/cdc/nbs/patient/profile/ethnicity/EthnicityEntrySteps.java

```diff
@@ -0,0 +1,35 @@
+package gov.cdc.nbs.patient.profile.ethnicity;
+
+import gov.cdc.nbs.testing.support.Active;
+import io.cucumber.java.en.Given;
+import java.util.List;
+import java.time.Instant;
+
+public class EthnicityEntrySteps {
+
+  private final Active<EthnicityDemographic> input;
+
+  EthnicityEntrySteps(final Active<EthnicityDemographic> input) {
+    this.input = input;
+  }
+
+  @Given("I am entering the ethnicity as of date {date}")
+  public void i_am_entering_patient_ethnicity_as_of(final Instant asOf) {
+    this.input.active(new EthnicityDemographic(asOf, null, null, null));
+  }
+
+  @Given("I enter the ethnicity unknown reason {string}")
+  public void i_enter_the_patient_ethnicity_unknown_reason(final String unknownReason) {
+    this.input.active(current -> current.withUnknownReason(unknownReason));
+  }
+
+  @Given("I enter the ethnicity ethnic group {string}")
```

Comment:
suggestion: Using `I enter the ethnicity ethnic group {ethnicity}` will let you write the step using values from the ethnic group value set like `Hispanic or Latino`, `Not Hispanic or Latino`, or `Unknown`.  The same applies to ethnicity_detailed, the cucumber parameter type for that value set is 'ethnicityDetail`